docs/standards/rest.md

```diff
@@ -115,6 +115,55 @@ headers:
 
 [Header field names are case insensitive](https://datatracker.ietf.org/doc/html/rfc7230#section-3.2). Snyk v3 API specs must use kebab case for consistency. All non-standard headers that are unique to Snyk must begin with `snyk-` (e.g. `snyk-requested-version`).
 
+## <a id="request-parameters"></a>Request Parameters
+
+### <a id="formats"></a>Formats
+
+Some resource types may be expressed with a media content-type format other than JSON API. These alternative formats may be offered with these format request parameters:
+
+#### `format` query parameter
+
+This is a URL query parameter of the form `?format=format_name`, where _format_name_ is an application-specific short nickname for the media content-type. This parameter's schema must be defined as an enum.
```

Comment:
Is "short nickname" important here, or is the idea to use whatever colloquial name exists for the format (whether it's short or not)? I understand you want to distinguish from the IANA-style content media type here but "short nickname" feels too ambiguous.